src/briefcase/commands/create.py

```diff
@@ -315,6 +326,11 @@ def install_app_support_package(self, app: BaseConfig):
             # the support package URL.
             try:
                 support_package_url = app.support_package
+                # Check if the URL mentions an unsupported architecture
+                url_parts = list(urlsplit(support_package_url))
+                for key, value in parse_qsl(url_parts[3]):
+                    if key == 'arch' and value != 'x86_64':
```

Comment:
This is a very specific check that will (a) break if you provide a custom support package URL, (b) only be true for Linux, and (c) could even change on Linux if someone wanted to submit a support packages for a different platform. For example, support for Raspbian isn't out of the question, which would mean arm64 support would be valid. Apple platforms already need to support arm64.

To that end, we should be looking for failures cloning the support package, and providing guidance on ways to fix the problem if the template can't be found, rather than pre-emptively assuming that a specific key and value will be present in the URL.

Reply:
Since a MissingNetworkResourceError would be raised when a support package is unavailable. Do you think the following change to the message would resolve the problem?

Current Message: 
   "Unable to download {url}; is the URL correct?"
New Message:
   "Unable to download {url}; is the URL correct? Suggestion : Compile the support package yourself, add a support_package reference to pyproject.toml file"